client/modules/workspace/src/SelectModal/SelectModal.tsx

```diff
@@ -8,18 +18,14 @@ import {
   TUser,
 } from '@client/hooks';
 
-import { BaseFileListingBreadcrumb } from '@client/common-components';
-import styles from './SelectModal.module.css';
 import {
+  BaseFileListingBreadcrumb,
   FileListingTable,
```

Comment:
Also outside scope: bug: underline highlight of file listing item extends to include icon
<img width="187" alt="Screenshot 2024-05-31 at 10 14 51 AM" src="https://github.com/DesignSafe-CI/portal/assets/20326896/d3dce2f0-781d-4a57-b45e-7fc39ffdd85f">

Reply:
good catch

Reply:
Subtask to handle all follow up items from this PR: [DES-2876](https://tacc-main.atlassian.net/browse/DES-2876)